src/tests/unit/tests/common/components/cards/__snapshots__/result-section-title.test.tsx.snap

```diff
@@ -1,976 +1,1966 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`ResultSectionTitle with outcomeType fail renders with alerting, badgeCount is 0 1`] = `
-<span
-  className="resultSectionTitle"
->
+<DocumentFragment>
   <span
-    className="screen-reader-only"
+    class="resultSectionTitle"
   >
-    test title
-     
     <span
-      role="alert"
+      class="screen-reader-only"
     >
-      0
-       
-      failures were
-       detected.
+      test title 
+      <span
+        role="alert"
+      >
+        0 failures were detected.
+      </span>
+    </span>
+    <span
+      aria-hidden="true"
+      class="title"
+    >
+      test title
+    </span>
+    <span
```

Comment:
This snapshot has a lot of extra HTML. I recommend mocking the `OutcomeChip` component so the snapshot is limited to relevant markup.

Reply:
Yes, we have mocked **OutcomeChip** component as per suggestion